bagitobjecttransfer/recordtransfer/views.py

```diff
@@ -169,11 +171,15 @@ def get_context_data(self, **kwargs) -> dict[str, Any]:
         context["groups_page_obj"] = groups_paginator.get_page(groups_page_number)
 
         context.update(
-            {
+            {   
+                # Form field element IDs
+                "ID_FIRST_NAME":  ID_FIRST_NAME,
```

Comment:
@danloveg 
I applied the same method of passing constants as context in this pull request. I know you proposed a cleaner way of passing context from template to JS by converting the python dictionary object into a JSON:

```
{{ js_context|json_script:"py_context_source_info" }}
```
This method does allow us to avoid defining each individual constant we want pass to the JS in the template. But I wonder if this is the best approach since we would have to dereference values using raw strings in the JS after parsing the JSON:

```javascript
// Context passed from template to JS
const sourceInfoContextElement = document.getElementById("py_context_source_info");
sourceInfoContext["id_source_type"] // 
```

Reply:
> We still end up having to use VARIABLE_NAME. If you change VARIABLE_NAME in the view, you have to update the Javascript to reflect the new name in both of those cases. The fact that one is in a string doesn't make a difference regarding maintainability.

I see, both a change in the variable name in the view (for the method of passing individual constants) and a change in the string key in the view would require a change in the JS in both cases.

In that case, I'll update the way I'm currently passing context to use your suggested method.

Reply:
Great, I will review this pull request after you make that change

Reply:
I've updated the way I'm passing context variables to be consistent with the new method in https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/281/commits/3de7ffd5e7c2d9d8791403762de1f02a8e4a08a5